### DISCUSSION TITLE
PostCSS config syntax

### DISCUSSION THREAD

**Discussion by blvdmitry** [postcss/postcss#1696] [General]
I haven't checked PostCSS documentation for a while and now have noticed that the only recommended config syntax is to use plugins with `require('plugin-name')`. However, some frameworks/bundlers do not support this syntax. For example, NextJS throws an error (https://github.com/vercel/next.js/issues/10117). 

Any plans or ideas on improving this experience? I can think that with arguments made by NextJS team in the issue it should be better to at least cover multiple syntax options. However, I don't know if it was removed intentionally from the docs earlier or not.

**Comment by ai** [comment]
We can move examples to object notation. Do you want to send PR?

**Comment by blvdmitry** [comment]
Sure, if you're ok with the change I will open a PR. Just got to check the exact syntax supported in code.
